onnxruntime/core/codegen/target/ort_tvm_utils.cc

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
What's the meaning of "target" in this folder structure? it seems contain many tvm ir building stuff, i didn't see the relationship with "target"

Reply:
target means target hardware ISAs, like ARM/x86/CUDA/etc. It is in line with the concept of target in TVM.

---
In reply to: [294446965](https://github.com/microsoft/onnxruntime/pull/888#discussion_r294446965) [](ancestors = 294446965)

Reply:
right, but i doesn't see any subfolder here for x86/arm/cuda/etc, and the contents in this folder looks like some ir builder / schedule for particualr target, it is very confusing.

---
In reply to: [297449201](https://github.com/microsoft/onnxruntime/pull/888#discussion_r297449201) [](ancestors = 297449201,294446965)

Reply:
It is just the first PR. :) Later, we will increase them.